readthedocs/rtd_tests/tests/test_views.py

```diff
@@ -168,3 +169,38 @@ def test_project_redirects(self):
     def test_project_redirects_delete(self):
         response = self.client.get('/dashboard/pip/redirects/delete/')
         self.assertRedirectToLogin(response)
+
+
+class SubprojectViewTests(TestCase):
+    def setUp(self):
+        self.user = N(User, username='test')
```

Comment:
really wish they had just used real words, though I guess we can do `import N as New`